.orchestra/config/components/revng_js_components.yml

```diff
@@ -0,0 +1,34 @@
+#@ load("@ytt:overlay", "overlay")
+#@ load("/lib/create_component.lib.yml", "single_build_component")
+
+#@yaml/text-templated-strings
+---
+#@ def js_components_args():
+configure: |
+  JS_DIR="$ORCHESTRA_DOTDIR/support/js-components"
+  mkdir -p "$BUILD_DIR"
+  cd "$BUILD_DIR"
+  cp "$JS_DIR/package.json" "$JS_DIR/yarn.lock" .
+  echo '170f9c4509efc5065364e99ac32823bf8a37fcbf1a05dd7cf85a530e4173d851  package.json' | sha256sum -c -
+  PACKAGE_URLS=$(grep 'resolved' yarn.lock | awk '{print $2}' | tr -d '"')
+  while IFS= read -r URL; do
+    wget -q "$URL"
```

Comment:
Can you split `.orchestra/support/extract.sh` into `fetch.sh` and `extract.sh` (which uses `fetch.sh`)?
This way we cache the archives in `SRC_ARCHIVE_DIR`.

Reply:
Done